Send files guidance for UI

saimaghafoor opened the pull request:
Content added for send files by email using the Notify website and the UI

To publish once UI send files by email feature is released.

saimaghafoor committed:
Send files guidance for UI

Content for sending files in the UI added in readiness for the feature release.

rparke commented:
> Thanks @rparke. Should I add it to the very top of the page (or you could add it as a suggestion in the right place and I'll accept?).

yeh just add it to the top of the file

saimaghafoor committed:
service link component added at the top 

Service link added at the top of the page to make service link work correctly

saimaghafoor committed:
Remove bulleted list

replaced bulleted list re security with a sentence instead as 2 lists close together may make it difficult to scan read/miss the short lead in sentence

saimaghafoor committed:
Bullet points tweaked

saimaghafoor committed:
Update send-files-by-email.html

Added content on link being encrypted

saimaghafoor committed:
Update guidance-send-files-by-email.html

sned files guidance for UI

saimaghafoor committed:
Update send-files-by-email.html

Small wording changes:
- security of file
- bullet points

saimaghafoor committed:
Small content change on how to send files

Content changes to instructions

quis committed:
Fix typo

quis committed:
move step into logical order

quis commented:
Before | After
---|---
<img width="779" height="322" alt="image" src="https://github.com/user-attachments/assets/1d668710-4745-44d2-9568-cef890336c89" /> | <img width="784" height="268" alt="image" src="https://github.com/user-attachments/assets/3bb770b9-1336-4564-bfdc-f08e694cfee4" />
<img width="765" height="807" alt="image" src="https://github.com/user-attachments/assets/cf1c1a4e-69a5-4b26-97ff-1cd1d72b4530" /> | <img width="764" height="730" alt="image" src="https://github.com/user-attachments/assets/5117623a-0650-4439-959d-465bc3aab735" />

quis commented:
I’ve changed the base of this branch so it will be merged into https://github.com/alphagov/notifications-admin/pull/5858

That means this can be merged now, and when we want to release we only have one PR to merge.
